ui/src/components/auth/LoginContainer.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import { signin } from '../../reducers/userSlice';
+import axios from 'axios';
+import { Box, Button, Checkbox, Link, Stack, TextField, Typography } from '@mui/material';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { z } from 'zod';
+import { LoginFormFields } from './helpers/form.types';
+import { formSpacing, formWidth } from './helpers/constants';
+
+// zod validation
+const loginValidationSchema = z.object({
+  email: z.string().email({ message: 'Please enter a valid email' }).trim(),
+  password: z
+    .string()
+    .trim()
+    .min(6, { message: 'Password must be at least 6 characters long' })
+    .max(100, { message: 'Password must be at most 100 characters long' }),
+});
+
+const LoginContainer: React.FC = () => {
+  const { register, handleSubmit, formState } = useForm<LoginFormFields>({
+    resolver: zodResolver(loginValidationSchema),
+  });
+
+  const { errors } = formState;
+
+  const [invalidLogin, setInvalidLogin] = useState('');
+  const [rememberPassword, setRememberPassword] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const handleEmailLogin: SubmitHandler<LoginFormFields> = async (data) => {
+    console.log(data);
+
+    try {
+      const payload = await axios.post('/api/auth/login', {
+        email: data.email,
+        password: data.password ? data.password : '',
+        isRememberPassword: rememberPassword,
+      });
+      dispatch(signin(payload.data));
+      setInvalidLogin('');
+      window.location.reload();
+    } catch (error: any) {
+      if (error?.response?.status === 500 || error?.response?.status === 401) {
+        setInvalidLogin('Invalid email or wrong password');
+      }
+      console.log(error);
+    }
+  };
+
+  return (
+    <>
+      <Box
+        minHeight={'100vh'}
+        sx={{
+          backgroundImage: `url(${process.env.PUBLIC_URL}/images/psalted-bg.jpg)`,
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          backgroundAttachment: 'fixed',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Box margin={'auto'} bgcolor={'transparent'} width={formWidth}>
+          <Stack direction={'column'} margin={'auto'} spacing={formSpacing}>
+            {/* <Box
```

Comment:
why you comment this out? if dont need can delete. if you wanna keep can explain in a comment, why we need to keep

Reply:
hahaha okok

Reply:
{/* <Box
              component="img"
              src={`${process.env.PUBLIC_URL}/images/ripple.png`}
              alt="Logo"
              height={{ xs: '4.5vh', sm: '5.5vh', md: '6.5vh', lg: '7.5vh', xl: '8.5vh' }}
              sx={{ filter: 'invert(0.6)' }}
              alignSelf={'center'}
            />
            <Typography
              variant="subtitle1"
              color="#656565"
              textAlign={'center'}
              fontWeight={'bold'}
            >
              Psalted 2.0
            </Typography> */}